### DISCUSSION TITLE
构建工程失败

### DISCUSSION THREAD

**Discussion by njuzhou** [CefView/QCefView#149] [Q&A] [answered]
问题描述:
构建NCW模式时，总是报错，报错信息如下:
mt.exe : general error c101008c: Failed to read the manifest from the resource of file "D:/Master/GIT/CEF/QCefView-1.0.6/build/CefViewCore/output/Release/bin/CefViewWing.exe"

尝试方案:
网上搜索后，已经尝试过，以管理员方式运行、cmake时去除勾选ATL和SANDBOX、工程中设置"生成清单"为否、ceftests目录下的cmakelist注释"add_windows_manifest."

请问大家是否有遇到过这种问题呢

**Comment by njuzhou** [comment, marked as answer]
已解决，最终发现最后提到的注释对应行可以解决问题

**Reply by tishion** [reply]
能否分享一下你是如何遇到这个问题的？似乎没有别人遇到过这个问题。

**Reply by njuzhou** [reply]
我也是按照网站教程上的进行操作，可能和本地什么因素相关吧。generate后正常进行cmake，但编译过程中，在第三步总是会构建出错，报错信息如问题中描述。
在网上搜索了相关信息，发现也有类似提问，解决方式有很多，一一尝试后，发现注释cmakelist中对应语句可以有效解决(因为直接跳过了这步，但应该不会对编译结果产生影响)

**Comment by L-Super** [comment]
我也遇到了此问题，后来将CefWing的inputresource注释掉之后就正常了
```cmake
# embed the manifest file
COMMAND mt.exe 
  -manifest \"${CMAKE_CURRENT_SOURCE_DIR}\\CefWing\\win\\CefViewWing.manifest\" 
  # -inputresource:\"$<TARGET_FILE:CefViewWing>\" 
  -outputresource:\"$<TARGET_FILE:CefViewWing>\"

```
